course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineScreen.kt

```diff
@@ -74,219 +53,144 @@ import org.openedx.core.domain.model.CoursewareAccess
 import org.openedx.core.extension.takeIfNotEmpty
 import org.openedx.core.presentation.course.CourseViewMode
 import org.openedx.core.ui.HandleUIMessage
-import org.openedx.core.ui.OfflineModeDialog
 import org.openedx.core.ui.OpenEdXButton
 import org.openedx.core.ui.TextIcon
 import org.openedx.core.ui.WindowSize
 import org.openedx.core.ui.WindowType
 import org.openedx.core.ui.displayCutoutForLandscape
-import org.openedx.core.ui.rememberWindowSize
 import org.openedx.core.ui.theme.OpenEdXTheme
 import org.openedx.core.ui.theme.appColors
 import org.openedx.core.ui.theme.appTypography
 import org.openedx.core.ui.windowSizeValue
-import org.openedx.course.DatesShiftedSnackBar
 import org.openedx.course.presentation.CourseRouter
-import org.openedx.course.presentation.container.CourseContainerFragment
-import org.openedx.course.presentation.container.CourseContainerTab
-import org.openedx.course.presentation.outline.CourseOutlineFragment.Companion.getUnitBlockIcon
 import org.openedx.course.presentation.ui.CourseDatesBanner
 import org.openedx.course.presentation.ui.CourseDatesBannerTablet
 import org.openedx.course.presentation.ui.CourseExpandableChapterCard
-import org.openedx.course.presentation.ui.CourseImageHeader
 import org.openedx.course.presentation.ui.CourseSectionCard
 import org.openedx.course.presentation.ui.CourseSubSectionItem
-import org.openedx.course.presentation.ui.DatesShiftedSnackBar
 import java.io.File
 import java.util.Date
+import org.openedx.course.R as CourseR
```

Comment:
This should be a default import without an alias.

Reply:
But we also have core resources there. Do you mean that we need to use the resources of the current module without aliases and add aliases for other resources? If I understand correctly, I will remove the alias for the course and add it for the core.